cyences_app_for_splunk/default/data/ui/views/cs_o365_reports.xml

```diff
@@ -1079,14 +1082,17 @@
       <table>
         <search>
           <query>`cs_o365` sourcetype="o365:management:activity" Workload IN (OneDrive, SharePoint) Operation IN ("AnonymousLinkUsed","SharingLinkUsed") NOT TargetUserOrGroupType IN ("Member", "SharePointGroup") $tkn_exclude_teams_link_accessed$ $tkn_application_name_link_accessed$
-|search UserId=anonymous
-| stats values(Operation) AS Operations, values(ObjectId) as ObjectId , values(ApplicationDisplayName) as ApplicationDisplayName, values(EventData) as EventData, values(ItemType) as ItemType,values(EventSource) as EventSource latest(_time) as LastTime earliest(_time) as FirstTime by SourceFileName, UserId 
-|rename UserId as UserAccessingFile
+| search UserId=anonymous
+| iplocation ClientIP  
+| fillnull value="" Country, Region, City 
+| eval Source = ClientIP 
```

Comment:
Doesn't seems to require "| eval Source = ClientIP".
you can use ClientIP or if you want to rename then use: | rename ClientIP as Source